tools/internal_ci/linux/grpc_sample_fuzzers.sh

```diff
@@ -15,13 +15,23 @@
 
 set -ex
 
+# avoid slow finalization after the script has exited.
```

Comment:
since the contents of this script is basically that of https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_bazel_rbe.sh, can't you simply use that script.
Also, it looks like what you're doing is basically trying to create one more of the bazel_rbe linux jobs, with a slightly different config. So it sounds like you should actually create one more job  (e.g. 
prod:grpc/core/master/linux/bazel_rbe/grpc_bazel_rbe_asan_fuzzer?) that is setup in exactly the same way as all the other bazel_rbe jobs, with only the config and bazel commandline being slightly different (you should still be able to call `grpc_bazel_rbe.sh`). 
That would make it much simpler to use the right flags/settings correctly, based on whether it's master/PR job (e.g. caching, bigquery test results uploading etc.) https://github.com/grpc/grpc/blob/3353d61c5afa98fd87a854aa1e21f42ee7d9adc0/tools/internal_ci/linux/grpc_bazel_rbe_asan.cfg#L37-L46